test/cutting/test_cutting_decomposition.py

```diff
@@ -284,3 +287,122 @@ def test_unused_qubits(self):
         )
         assert subcircuits.keys() == {"A", "B"}
         assert subobservables.keys() == {"A", "B"}
+
+    def test_generate_cutting_experiments(self):
```

Comment:
I wonder if `generate_cutting_experiments` should go in its own file, instead of `cutting_decomposition.py`, perhaps a name like `cutting_experiments.py`.  I agree that if, at a high level, we think of our problem as 1. decompose 2. execute 3. reconstruct, then this is part of the "decompose" step.  But it's a bit logically distinct from the existing code in `cutting_decomposition.py`, and it would likely help with our organization to keep it separated.  They're really two distinct modules/steps within the decomposition.

Anyway, the reason I commented on this line is because if we do this, the new tests should be placed in something like `test_cutting_[sub]experiments.py` instead.

Reply:
Ya I'm open to this. I don't think it's a blocker for this PR. When we move the function we'll make a decision in that PR. I'm a bit on the fence right now, but I'd like to hear more of your reasoning.

Reply:
I've moved the test to test_evaluation until we decide on where it all should live. This also cleans up the diff in this PR a bit